src/agents/common/actor.py

```diff
@@ -18,5 +18,11 @@ def __init__(self, state_dim, action_dim, max_action):
             nn.Tanh()
         )
 
+        # Initialize weights
```

Comment:
Agents might want to initialize the weights differently - might be an idea to move this out of the `common/actor.py` file and put it into the agent initialization directly?

This would allow us to be more flexible in future when we want to copy agents from previous runs.